packages/flutter_rx_bloc/pubspec.yaml

```diff
@@ -12,7 +12,8 @@ dependencies:
   flutter:
     sdk: flutter
   provider: ^5.0.0
-  rx_bloc: ^2.0.0
+  rx_bloc:
```

Comment:
@DDavidPrime, I believe we need to have a specific version of rx_bloc

Reply:
Updated!